cmd/cmd.go

```diff
@@ -73,13 +72,17 @@ func NewPackCommand(logger ConfigurableLogger) (*cobra.Command, error) {
 	rootCmd.AddCommand(commands.SetDefaultBuilder(logger, cfg, &packClient))
 	rootCmd.AddCommand(commands.InspectBuilder(logger, cfg, &packClient, writer.NewFactory()))
 	rootCmd.AddCommand(commands.SuggestBuilders(logger, &packClient))
+	//nolint:staticcheck
```

Comment:
Out of curiosity, what lint warning are we suppressing?

Reply:
Using a deprecated command.